text/0005-white-flag/0005-white-flag.md

```diff
@@ -160,8 +153,8 @@ by allowing a potential increase of TPS/CTPS.
 
 A node consumes and produces snapshot files and bases the computation of its ledger state on them. In the current
 network, if one of these files was tampered with and fed to a node, it would eventually lead to an invalid ledger state
-where a bundle confirmed by a milestone would actually be a double spend. This situation would be detected by the node
-and it would stop its activities as a security measure. However, with White Flag, such bundles would be confirmed by
+where a message confirmed by a milestone would actually be a double spend. This situation would be detected by the node
+and it would stop its activities as a security measure. However, with White Flag, such messages would be confirmed by
 milestones but ignored by the node, the fake snapshot then going unnoticed. The ledger state would then become more and
 more corrupted and the view of the balances completely wrong, errors just accumulating over time. The need for a
 snapshot verification mechanism is then amplified by the implementation of White Flag. This mechanism being out of the
```

Comment:
```suggestion
snapshot verification mechanism is then amplified by the implementation of White Flag. This mechanism being out of the [unfinished sentence!]
```

Reply:
The sentence is complete, it's just Github annoying GUI, you have to click on the little arrow.